web/ibexvis.js

```diff
@@ -0,0 +1,66 @@
+console.log("Initialising")
+
+import "https://cdn.plot.ly/plotly-3.2.0.min.js";
+import "https://cdn.jsdelivr.net/pyodide/v0.29.0/full/pyodide.js";
+
+console.log("Loaded pyodide and plotly")
+
+let pyodide = await loadPyodide();
+await pyodide.loadPackage(["numpy", "matplotlib", "micropip"]);
+const micropip = pyodide.pyimport("micropip");
+await micropip.install("emfs:///ibex_vis-0.0.1-py3-none-any.whl");
```

Comment:
@oerc0122 we need to zip the contents of the `<root>/ibex_vis` folder as a "wheel" for this line to work... I'll figure out the code...

Reply:
It is compiled into a wheel as part of the CI and moved into the web folder.